src/Config/ConfigWriter.php

```diff
@@ -84,16 +84,27 @@ protected function parseContentValue($contents, $path, $value)
 
         $count = 0;
         $patterns = array();
+
+        // this should be first
```

Comment:
Either explain this comment more thoroughly or remove it.

Reply:
already update